content/language/dotnet/configure-ci-cd.md

```diff
@@ -107,6 +107,7 @@ to Docker Hub.
            uses: docker/build-push-action@v5
            with:
              context: .
+             platforms: linux/amd64,linux/arm64/v8
```

Comment:
The `/v8` suffix is not necessary for arm64
https://github.com/containerd/containerd/blob/aa8b094ab1b6b345897b4ddb44bc3f249cc9f0f7/platforms/platforms.go#L105

```suggestion
             platforms: linux/amd64,linux/arm64
```